digits/scheduler.py

```diff
@@ -175,6 +177,30 @@ def add_job(self, job):
             return False
         else:
             self.jobs.append(job)
+
+            if flask._app_ctx_stack.top is not None:
```

Comment:
File "/home/jmancewicz/dev/digits/DIGITS/digits/templates/job_row.html", line 5, in top-level template code
    <td><h4 class="list-group-item-heading"><a href="{{ url_for(show_func, job_id=job.id()) }}">{{ job.name() }}  </a></h4></td>
  File "/usr/local/lib/python2.7/dist-packages/flask/helpers.py", line 287, in url_for
    raise RuntimeError('Application was not able to create a URL '

@lukeyeager, I think this is close to the error that caused me to add the line you questioned. url_for doesn't work in this test.  From what I can see it's that the server is not running or the SERVER_NAME is not set.  It feels like we talked about this, but I don't recall if there was a resolution.